modules/simple_forms_api/app/services/simple_forms_api/form_upload_notification_email.rb

```diff
@@ -48,6 +48,10 @@ def check_missing_keys(config)
       all_keys = %i[form_data form_number date_submitted confirmation_number]
 
       missing_keys = all_keys.select { |key| config[key].nil? || config[key].to_s.strip.empty? }
+      email = config.dig(:form_data, :email)
+      first_name = config.dig(:form_data, :full_name, :first)
+      missing_keys << 'form_data: email' if email.nil? || email.to_s.strip.empty?
+      missing_keys << 'form_data: first_name' if first_name.nil? || first_name.to_s.strip.empty?
```

Comment:
`first_name.blank? `checks for nil, empty strings, and whitespace strings.

Reply:
Oh awesome, I will fix that!